swim/member.go

```diff
@@ -123,11 +129,27 @@ type Change struct {
 	Address           string `json:"address"`
 	Incarnation       int64  `json:"incarnationNumber"`
 	Status            string `json:"status"`
+	Tombstone         bool   `json:"tombstone,omitempty"`
```

Comment:
I don't think we need to cover such cases where the standard library offers us this capability. There is no functional change if it happens to fail, it is just an optimization.